Makefile

```diff
@@ -7,7 +7,7 @@ PROJECT_REPO := github.com/crossplane/$(PROJECT_NAME)
 PLATFORMS ?= linux_amd64 linux_arm64
 
 CODE_GENERATOR_COMMIT ?= 285d87b66b62fbfb859986ddf74c9f9b6ae743fb
-GENERATED_SERVICES="apigatewayv2,athena,cloudfront,cloudwatchlogs,dynamodb,elbv2,ec2,efs,glue,iot,kafka,kms,lambda,mq,rds,secretsmanager,servicediscovery,sfn,transfer,ram"
+GENERATED_SERVICES := $(shell find ./apis/ -type f -name generator-config.yaml -regextype posix-extended  -not -regex  ".*/(eks|docdb|route53resolver)/.*" | cut -d/ -f 3 | tr '\n' ' ')
```

Comment:
Why do we need to skip those services?

Reply:
today we missing eks, route53resolver and docdb in makefile - 
 check:
 https://github.com/crossplane/provider-aws/pull/1006
https://github.com/crossplane/provider-aws/pull/1004
https://github.com/crossplane/provider-aws/pull/1003

Reply:
#1006 #1004 #1003 fixed